[Bug]: Lighthouse CI Error - Cannot find module 'decamelize'

## Describe the bug
Lighthouse CI job fails with "Cannot find module 'decamelize'" error when running in GitHub Actions. The error occurs in the Lighthouse CI tools during the `yargs-parser` module initialization.

## To Reproduce
Steps to reproduce the behavior:
1. Create a Pull Request in the RMS repository
2. GitHub Actions workflow triggers the Lighthouse CI job
3. The job fails with MODULE_NOT_FOUND error for 'decamelize' module
4. See error in GitHub Actions logs

## Expected behavior
Lighthouse CI should run successfully and generate a performance audit report without module dependency errors.

## Screenshots
N/A - Error is in GitHub Actions logs

## Desktop (please complete the following information):
- OS: GitHub Actions (Ubuntu 24.04.2 LTS)
- Environment: GitHub Actions Runner
- Node.js: v20.19.5

## Additional context
Error occurs in the Lighthouse CI dependency chain:
```
Error: Cannot find module 'decamelize'
Require stack:
- /home/runner/work/RMS/RMS/node_modules/@lhci/cli/node_modules/yargs/node_modules/yargs-parser/index.js
- /home/runner/work/RMS/RMS/node_modules/@lhci/cli/node_modules/yargs/build/lib/command.js
- /home/runner/work/RMS/RMS/node_modules/@lhci/cli/node_modules/yargs/build/lib/yargs.js
- /home/runner/work/RMS/RMS/node_modules/@lhci/cli/node_modules/yargs/yargs.js
- /home/runner/work/RMS/RMS/node_modules/@lhci/cli/node_modules/yargs/index.js
- /home/runner/work/RMS/RMS/node_modules/@lhci/cli/src/cli.js
```

This appears to be a dependency issue where the `decamelize` module is missing from the Lighthouse CI installation. The error occurs during the restoration of Lighthouse CI tools from cache.

--- a/.github/workflows/lighthouse-report.yaml
+++ b/.github/workflows/lighthouse-report.yaml
@@ -152,32 +152,16 @@ jobs:
             build-
           fail-on-cache-miss: true
 
-      - name: Cache Lighthouse CI tools
-        id: cache-lighthouse-tools
-        uses: actions/cache@v4
-        with:
-          path: |
-            ~/.npm
-            node_modules/@lhci
-            node_modules/wait-on
-            node_modules/which-module
-            node_modules/.bin/lhci
-            node_modules/.bin/wait-on
-          key: ${{ runner.os }}-lighthouse-tools-${{ hashFiles('**/package-lock.json') }}-v2
-          restore-keys: |
-            ${{ runner.os }}-lighthouse-tools-v2-
+      - name: Install dependencies (if not cached)
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: npm ci --prefer-offline
 
       - name: Install Lighthouse CI tools
-        if: steps.cache-lighthouse-tools.outputs.cache-hit != 'true'
-        run: npm install --no-save @lhci/cli wait-on which-module
+        run: npm install --no-save @lhci/cli@0.12.x wait-on
 
       - name: Verify Lighthouse CI tools
         run: |
-          if [ "${{ steps.cache-lighthouse-tools.outputs.cache-hit }}" == "true" ]; then
-            echo "✓ Lighthouse CI tools restored from cache"
-          else
-            echo "✓ Lighthouse CI tools installed"
-          fi
+          echo "✓ Lighthouse CI tools installed"
           npx lhci --version
           npx wait-on --version
 

